devel/scanitemother.lua

```diff
@@ -0,0 +1,20 @@
+-- Display the item lists that the selected item is part of.
+local selected_item = dfhack.gui.getAnyItem()
+
+if not selected_item then
+    qerror("This script requires an item selected!")
+end
+
+local indices = {}
+for index, value in pairs(df.global.world.items.other) do
+    for _, item in pairs(value) do
+        if item == selected_item then
+            table.insert(indices, index)
+        end
+    end
+end
+
+print("Found in the following indecies:")
+for _, index in pairs(indices) do
+    print(" " .. index)
```

Comment:
could this print out both the numeric index and the *name* of the OTHER list? that would make it a little more handy.

Reply:
Do you mean the numeric index the index appears in `df.global.world.items.other` at or just the numeric index of our local table? Currently the output is:

```
Found in the following indecies:
 IN_PLAY
 ANY_GENERIC36
 CLOTH
```

Reply:
I'd say that's good. Thanks!